src/lib.rs

```diff
@@ -182,30 +132,63 @@ impl<C: Config> Plugin for GgrsPlugin<C> {
             ..default()
         });
 
-        app.init_resource::<RollbackTypeRegistry>()
-            .init_resource::<RollbackSnapshots>()
-            .init_resource::<RollbackFrameCount>()
+        app.init_resource::<RollbackFrameCount>()
+            .init_resource::<RollbackFrameConfirmed>()
             .init_resource::<LocalPlayers>()
             .init_resource::<FixedTimestepData>()
             .add_schedule(GgrsSchedule, schedule)
             .add_schedule(ReadInputs, Schedule::new())
             .add_systems(PreUpdate, schedule_systems::run_ggrs_schedules::<C>)
-            .add_systems(LoadWorld, load_world)
-            .add_systems(SaveWorld, save_world);
+            .add_plugins((
+                GgrsSnapshotSetPlugin,
+                GgrsChecksumPlugin,
+                GgrsResourceSnapshotCopyPlugin::<Checksum>::default(),
```

Comment:
There is no guarantee this will run after the checksumming plugins, so if `GgrsResourceSnapshotCopyPlugin::save` runs before `GgrsChecksumPlugin::update` (which it does for me at least), we'll end up saving an outdated checksum in `handle_requests`.

Reply:
Good spot! I didn't notice that at all, you're right in that there should be an explicit `Checksum` set to ensure the right checksum is saved.